src/pkg/leaderElection.go

```diff
@@ -43,8 +44,34 @@ func RunLeaderElection(client *clientset.Clientset, lockName, lockIdentity, lock
 				log.Info().Msgf("Perform Migration")
 				for {
```

Comment:
This actually will be a good place to use a generated logger.

If you go look at https://github.com/OpsLevel/opslevel-runner/blob/main/src/cmd/run.go#L159

you will see
```
logger := log.With().Int("worker", 0).Logger()
```
and then we do
```
logger.Info().Msg("Stopped Polling for jobs ...")
```

This is how you build up context on a logger that gets reused.

Up around line 33 outside of the `RunOrDie` we could setup a logger for the leader

```
logger := log.With().String("worker", "leader").Logger()
```

This will add context to our log lines that they are come from the "leader" go routine.